homeassistant/components/fritz/device_tracker.py

```diff
@@ -79,6 +79,14 @@ def get_device_name(self, device):
             return None
         return ret
 
+    def get_extra_attributes(self, device):
+        """Return the name of the given device or None if is not known."""
```

Comment:
Please update the docstring.

Reply:
Thanks. I totally forgot. Its now updated